extensions/interactions/NumberWithUnits/directives/number_with_units_help_modal_directive.html

```diff
@@ -0,0 +1,149 @@
+<div class="modal-header">
+  <b>Guidelines for writing units</b>
+</div>
+<div class="modal-body">
+  <style>
+    td {
+      padding: 5px;
+      width: 30%;
+    }
+  </style>
+  <table class="table-bordered">
+    <tr>
+      <td><b>Rule Description</b></td>
+      <td><b>Incorrect Format</b></td>
+      <td><b>Correct Format</b></td>
+    </tr>
+    <tr>
+      <td>
+        Separate value and the unit with one space.
+      </td>
+      <td>3km</td>
+      <td>3 km</td>
+    </tr>
+    <tr>
+      <td>
+        In case the value is a fraction, separate numerator and denominator by
+        forward slash '/' symbol.
+      </td>
+      <td>3 by 5 km</td>
+      <td>3/5 km</td>
+    </tr>
+    <tr>
+      <td>
+        For multiplying two or more units, put one space or asterisk '*' between them.
+      </td>
+      <td>5 J-s <br>  5 J . s</td>
+      <td>5 J s <br> 5 J * s</td>
+    </tr>
+    <tr>
+      <td>
+        For adding dimensions to the unit, use caret '^' symbol to add both positive
+        and negative powers.
+      </td>
+      <td>5 kg / m(2) <br> 5 kg / m**2 <br> 5 kg m^(-2)</td>
+      <td>5 kg / m^2 <br> 5 kg / m^2 <br> 5 kg m^-2</td>
+    </tr>
+    <tr>
+      <td>
+        In case of multiple units in the denominator, enclose them within the parenthesis.
+      </td>
+      <td>5 kg / m * s <br> 5 kg/m*s</td>
+      <td>5 kg / (m * s) <br> 5 kg / m / s</td>
+    </tr>
+    <tr>
+      <td>
+        For writing beginning units (like currencies), separate unit and value with one space.
+      </td>
+      <td>$5 <br> 5$ <br> 5 $</td>
+      <td>$ 5</td>
+    </tr>
+  </table><br><br>
+
+  <table class="table-bordered">
+    <tr>
+      <td><b>Base</b></td>
+      <td><b>Unit</b></td>
+    </tr>
+    <tr>
+      <td>Length</td>
+      <td>meter (m), inch (in), foot (ft), yard (yd), mile (mi), link (li), rod (rd), chain (ch), angstrom, mil</td>
+    </tr>
+    <tr>
+      <td>Surface area</td>
+      <td>m2, sqin, sqft, sqyd, sqmi, sqrd, sqch, sqmil, acre, hectare</td>
+    </tr>
+    <tr>
+      <td>Volume</td>
+      <td>m3, litre (l, L, lt, liter), cc, cuin, cuft, cuyd, teaspoon, tablespoon</td>
+    </tr>
+    <tr>
+      <td>Angles</td>
+      <td>rad (radian), deg (degree), grad (gradian), cycle, arcsec (arcsecond), arcmin (arcminute)</td>
+    </tr>
+    <tr>
+      <td>Time</td>
+      <td>
+        second (s, secs, seconds), minute (mins, minutes), hour (h, hr, hrs, hours), day (days),
+        week (weeks), month (months), year (years), decade (decades), century (centuries), millennium (millennia)
+      </td>
+    </tr>
+    <tr>
+      <td>Frequency</td>
+      <td>hertz (Hz)</td>
+    </tr>
+    <tr>
+      <td>Mass</td>
+      <td>
+        gram(g), tonne, ton, grain (gr), dram (dr), ounce (oz), poundmass (lbm, lb, lbs),
```

Comment:
kilogram? (mentioning this since it's an SI standard unit)

Reply:
Done